src/applications/mhv-secure-messaging/tests/e2e/fixtures/multi-facilities-some-blocked-recipients-response.json

```diff
@@ -0,0 +1,128 @@
+        "relationType": "PATIENT",
+        "preferredTeam": true,
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "6832726",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 6832726,
+        "name": "TG-7410",
+        "stationNumber": "531",
+        "blockedStatus": true,
+        "relationType": "PATIENT",
+        "preferredTeam": true,
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "6910405",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 6910405,
+        "name": "Jeasmitha-Cardio-Clinic",
+        "stationNumber": "531",
+        "blockedStatus": false,
+        "relationType": "PATIENT",
+        "preferredTeam": true,
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "6978853",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 6978853,
+        "stationNumber": "583",
+        "blockedStatus": false,
+        "name": "***TG 100_SLC4%",
+        "relationType": "PATIENT",
+        "preferredTeam": true,
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "6978854",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 6978854,
+        "stationNumber": "583",
+        "blockedStatus": false,
+        "name": "***TG 200_APPT_SLC4%",
+        "relationType": "PATIENT",
+        "preferredTeam": true,
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "7123233",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 7123233,
+        "name": "CAMRY_PCMM RELATIONSHIP_05092022_SLC4",
+        "stationNumber": "583",
+        "blockedStatus": false,
+        "relationType": "PATIENT",
+        "preferredTeam": true,
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "56219",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 56219,
+        "stationNumber": "583",
+        "blockedStatus": false,
+        "name": "SLC4 PCMM",
+        "relationType": "PATIENT",
+        "preferredTeam": true,
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "3658288",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 3658288,
+        "name": "Record Amendment Admin",
+        "stationNumber": "531",
+        "blockedStatus": false,
+        "preferredTeam": true,
+        "relationshipType": "PATIENT",
+        "ohTriageGroup": false
+      }
+    },
+    {
+      "id": "3658299",
+      "type": "all_triage_teams",
+      "attributes": {
+        "triageTeamId": 3658299,
+        "name": "OH TG GROUP 002",
+        "stationNumber": "531",
+        "blockedStatus": false,
+        "preferredTeam": true,
+        "relationshipType": "PATIENT",
+        "ohTriageGroup": true
+      }
+    }
+  ],
+  "meta": {
+    "associatedTriageGroups": 7,
+    "associatedBlockedTriageGroups": 0,
```

Comment:
same, I see some objects have blocked status and that probably should be reflected here